airbyte-cdk/python/airbyte_cdk/sources/declarative/stream_slicers/stream_slicer.py

```diff
@@ -28,3 +28,45 @@ def stream_slices(self) -> Iterable[StreamSlice]:
 
         :return: List of stream slices
         """
+
+    def set_parent_state(self, stream_state: StreamState) -> None:
```

Comment:
let's leave the method as abstract to force the concrete class to implement it (or intentionally implement a no-op)

Reply:
Fixed